docs/topics/data-metadata-management/index.md

```diff
@@ -19,52 +19,54 @@ Welcome! If you are involved in data or metadata standards aspects of establishi
+| Beacon | Supports discovery of genomic variants and individuals. | v1.0.1 | v0.3 | <a href="https://github.com/ga4gh-beacon/specification-v2" target="_blank">Specification</a>, <a href="https://ega-archive.org/beacon/#/" target="_blank">Documentation</a>, <a href="https://ega-archive.org/beacon-api/" target="_blank">Endpoint</a> | N/A |
+| Crypt4GH | Enables direct byte-level compatible random access to encrypted genetic data stored in community standards (*e.g.* CRAM, VCF). | v1.0 | v1.0 | <a href="http://samtools.github.io/hts-specs/crypt4gh.pdf" target="_blank">Specification</a>, <a href="https://github.com/EGA-archive/crypt4gh" target="_blank">Documentation</a>, Endpoint | <a href="https://doi.org/10.1093/bioinformatics/btab087" target="_blank">DOI</a> |
+| Data Use Ontology (DUO) | Allows users to semantically tag datasets with usage restrictions so datasets can be automatically discoverable based on a researcher's authorization level or intended use. | 2021-02-23 | 2021-02-23 | <a href="https://github.com/EBISPOT/DUO" target="_blank">Specification</a>, <a href="https://ega-archive.org/data-use-conditions" target="_blank">Documentation</a>, <a href="https://www.ebi.ac.uk/ols/api/ontologies" target="_blank">Endpoint</a> | <a href="https://doi.org/10.1016/j.xgen.2021.100028" target="_blank">DOI</a> |
+| htsget | Enables secure, efficient, and reliable access to sequencing read and variation data including specific genomic regions. | v1.3.0 | v1.0.0 | <a href="http://samtools.github.io/hts-specs/htsget.html" target="_blank">Specification</a>, <a href="https://github.com/EGA-archive/ega-download-client" target="_blank">Documentation</a>, <a href="https://ega.ebi.ac.uk:8052/elixir/tickets/tickets" target="_blank">Endpoint</a> | <a href="https://doi.org/10.1093/bioinformatics/bty492" target="_blank">DOI</a> |
+| refget | Enables access to reference sequences using an identifier derived from the sequence itself. | v1.2.6 | N/A | <a href="http://samtools.github.io/hts-specs/refget.html" target="_blank">Specification</a>, Documentation, Endpoint | <a href="https://doi.org/10.1093/bioinformatics/btab524" target="_blank">DOI</a> |
+| Researcher IDs (Passports, Visas) | Specifies the collection of researchers that may access a dataset at any given time, and the credentials they must supply. | v1.0.1 | v1.0.1 | <a href="https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md" target="_blank">Specification</a>, <a href="https://docs.google.com/document/d/1FTzUYAfV5d2a0zoDkbY9Iy_L5NbSAnHeWnmY2NIrY8M/edit" target="_blank">Documentation</a>, <a href="https://ega.ebi.ac.uk:8443/ega-permissions/swagger-ui/index.html" target="_blank">Endpoint</a> | <a href="https://doi.org/10.1016/j.xgen.2021.100030" target="_blank">DOI</a> |
```

Comment:
Out of curiosity, why have Visas in uppercase? I've seen the DOI where they put the "passport" specification with uppercase, but not visas. And wouldn't it be "Passports and visas"?

Reply:
I just wanted to highlight line 49, but git highlighted all these. Sorry about that 😃

Reply:
Changed to "visas"